tools/crmadmin.c

```diff
@@ -89,7 +79,7 @@ static GOptionEntry command_options[] = {
 };
 
 static GOptionEntry additional_options[] = {
-    { "timeout", 't', 0, G_OPTION_ARG_INT, &options.timeout,
+    { "timeout", 't', 0, G_OPTION_ARG_STRING, &options.timeout,
       "Time (in milliseconds) to wait before declaring the"
```

Comment:
This help string needs updating too. Looking at other commands with --timeout, we're not consistent about how to describe it, but the one in crm_resource is decent:

    Abort if command does not finish in this time

crm_mon has a couple of other options that take time specifications, it uses TIMESPEC for the argument description and then has this:

    The TIMESPEC in any command line option can be specified in many different
    formats.  It can be just an integer number of seconds, a number plus units
    (ms/msec/us/usec/s/sec/m/min/h/hr), or an ISO 8601 period specification.

Something like either of those would be fine.